.github/workflows/prettier-check.yml

```diff
@@ -0,0 +1,31 @@
+name: autofix.ci
+on:
+  push:
+    branches:
+      - trunk
+    paths:
+      - "**.html"
+      - "**.css"
+      - "**.js"
+      - "**.md"
+      - "**.tsx"
+      - "**.json"
+  pull_request:
+    branches:
+      - trunk
+  merge_group:
+    types: [checks_requested]
+  page_build:
```

Comment:
in case we ever change the published github pages branch
https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#page_build